public/src/MockGRPCManager.ts

```diff
@@ -457,57 +455,51 @@ export class MockGrpcManager {
 
     public getSubmissionsByCourse(courseID: bigint, type: SubmissionRequest_SubmissionType): Promise<IGrpcResponse<CourseSubmissions>> {
         // TODO: Remove `.clone()` when done migrating to AsObject in state
```

Comment:
Can this be removed? There are still some `.clone()` calls, but not sure if those are related to this. Moreover, there are no more `AsObject`.